client/campaign_message_configuration/README.md

```diff
@@ -0,0 +1,79 @@
+# Campaign Message Configuration
+
+Store campaign message configurations for their respective campaigns. This allows users to switch between campaigns while keeping each of the message configurations and restoring them when the user returns to the original campaign. New campaigns can either be created with customizable default
+settings or from the existing configuration (see the "Transfer Settings" option).
+
+## Getting Started
+
+These instructions will get you a copy of the plugin up and running on your local machine for development and testing purposes. 
+
+### Prerequisites
+
+What you need to install the software:
+
+```
+Linux or Windows 
+King Phisher Minimum Required Version = 1.10.0 
+```
+
+### Installing 
+
+- [ ] Download plugin 
+- [ ] Move plugin to: <code>/opt/king-phisher-plugins/client/</code>
+- [ ] Configure plugin<code>(refer to Deployment & Configuration below)</code> 
+
+## Running the tests
+
+In order to test if `campaign_message_configuration.py` is operational, we must change campaigns then return back to the original campaign. To create a new campaign, navigate to the King Phisher Client. Once inside of the client we need to open the campaigns menu, to do this go to `File -> Open Campaign -> New Campaign. Once you have generated a new campaign, revert back to the original campaign that you created and check that your settings, etc.. If all campaign information is being saved and properly carried over then the plugin is working properly!  
+
+>It is recommended that end users implementing this plugin run the King Phisher Client in debug mode before testing the plugin. `DEBUG` is accomplished by inserting `-L DEBUG` onto the execution command: 
+```
+# CLIENT EXAMPLE 
+testing@testVM:/opt/king-phisher$ ./KingPhisher -L DEBUG

```

Comment:
remove the test@testVM part and just rap the command in `

Reply:
Also how to start the King Phisher Client does not need to be in the plugin readme documentation.